pkg/authorization/interfaces.go

```diff
@@ -0,0 +1,31 @@
+package authorization
+
+import (
+	"net/http"
+
+	"github.com/oauth2-proxy/oauth2-proxy/pkg/apis/sessions"
+)
+
+// Policy is a Rule policy name
+type Policy string
+
+const (
+	AllowPolicy = Policy("ALLOW")
```

Comment:
Is it better to initialise these as below, looks less like we are calling some function this way 
```suggestion
	AllowPolicy Policy = "ALLOW"
```